nsqd/topic.go

```diff
@@ -275,6 +275,7 @@ func (t *Topic) messagePump() {
 				chanMsg.deferred = msg.deferred
 			}
 			if chanMsg.deferred != 0 {
+				atomic.AddUint64(&channel.messageCount, 1)
```

Comment:
I don't think we want this change, though, because when the deferred message is finally published this will be incremented.